src/blueprints/tv2/factories/tv2-atem-video-switcher-timeline-factory.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Tv2VideoSwitcherTimelineObjectFactory
+} from '../value-objects/factories/tv2-video-switcher-timeline-object-factory'
+import { Tv2DownstreamKeyer } from '../value-objects/tv2-studio-blueprint-configuration'
+import { AtemDownstreamKeyerTimelineObject, AtemType } from '../../timeline-state-resolver-types/atem-types'
+import { Tv2AtemLayer } from '../value-objects/tv2-layers'
+import { DeviceType } from '../../../model/enums/device-type'
+
+export class Tv2AtemVideoSwitcherTimelineFactory implements Tv2VideoSwitcherTimelineObjectFactory {
+  public createDownstreamKeyerTimelineObject(downstreamKeyer: Tv2DownstreamKeyer, layer: string, onAir: boolean): AtemDownstreamKeyerTimelineObject {
+    return {
+      id: '',
+      enable: {
+        while: 1
+      },
+      priority: 10,
+      layer,
+      content: {
+        deviceType: DeviceType.ATEM,
+        type: AtemType.DSK,
+        dsk: {
+          onAir,
+          sources: {
+            fillSource: downstreamKeyer.Fill,
+            cutSource: downstreamKeyer.Key
+          },
+          properties: {
+            clip: downstreamKeyer.Clip * 10, // input is percents (0-100), atem uses 1-000
+            gain: downstreamKeyer.Gain * 10, // input is percents (0-100), atem uses 1-000
```

Comment:
Instead of the comment you could have a method called something like `convertPercentageToAtemPercentageValue()` so it would be:
```
clip: this.convertPercentageToAtemPercentageValue(downstreamKeyeer.Clip),
gain: this.convertPercentageToAtemPercentageValue(downstreamKeyeer.Gain)
```

Also: I believe you intented to write `1 - 1000` and not `1 - 000`?

Reply:
Created and changed to use the suggested method.

The comment is dirrectly copied from Blueprint, and i think originally written by Krzy.
Should i add the missing 1?

Reply:
> Should i add the missing 1?

Preferable we wouldn't need the comment at all.